properties-workitem/src/test/resources/org.jbpm.process.workitem.properties.properties_poc.properties-loader.v1.0.bpmn2

```diff
@@ -0,0 +1,144 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This process was supposed to be used in a test or something else?

Reply:
Deleted, test class included in newest commit

Reply:
Test class is still not here.